docs/specification/1.0.md

```diff
@@ -97,6 +97,33 @@ Code | Description
 
 ## Calling a CDS Service
 
+### HTTP Request
+
+An EHR calls a CDS service by `POST`ing a JSON document to the service
+endpoint, which can be constructed from the CDS Service base URL and an
+individual service id as `{baseUrl}/cds-services/{service.id}`.  The CDS Hook
+call includes a JSON POST body with the following input fields:
+
+Field | Priority | Description
+----- | ----- | --------
+`hook` | REQUIRED | *string* or *URL*. The hook that triggered this CDS Service call<br />(todo: link to hook documentation)
+<nobr>`hookInstance`</nobr> | REQUIRED | *string*.  A UUID for this particular hook call (see more information below)
+`fhirServer` | OPTIONAL | *URL*.  The base URL EHR's [FHIR](https://www.hl7.org/fhir/) server. If fhirAuthorization is provided, this field is REQUIRED.  The scheme should be `https`
+`fhirAuthorization` | OPTIONAL | *object*. A structure holding an OAuth 2.0 bearer access token granting the CDS Service access to FHIR resources, along with supplemental information relating to the token. See the [FHIR Resource Access](#fhir-resource-access) section for more information.
+`user` | REQUIRED | *string*.  The FHIR resource type + id representing the current user.<br />The type is one of: [Practitioner](https://www.hl7.org/fhir/practitioner.html), [Patient](https://www.hl7.org/fhir/patient.html), or [RelatedPerson](https://www.hl7.org/fhir/relatedperson.html).<br />For example, `Practitioner/123`
+`patient` | REQUIRED | *string*.  The FHIR `Patient.id` of the current patient in context
```

Comment:
Are you sure `patient` and `encounter` are REQUIRED?  There may be scenarios where there might not be a specific patient in context.

Reply:
@olbrich, it did occur to me that some queries may not be patient-specific.  @kpshek, do you agree?

Reply:
Recommend holding until decision is made on #119

Reply:
@bakerdb - As @mattberther mentioned, we are going to move `patient` and `encounter` to context in #119. We are doing this because both `patient` and `encounter` are optional for hooks that are not patient/encounter specific.

In order to keep this PR moving, I'm going to merge this in but change `encounter` to `OPTIONAL`. I will have a PR for #119 later today that removes both `patient` and `encounter` out and into context for the hooks where appropriate.